src/octopeer-github/main/Status.ts

```diff
@@ -10,6 +10,12 @@ enum StatusCode {ERROR, RUNNING, OFF, STANDBY}
 const Status = new (class Status {
 
     /**
+     * Stores the current status.
+     * @type {StatusCode}
+     */
+    private status: StatusCode = StatusCode.OFF;
```

Comment:
Default status must be standby. Because the default option of logging is true (see `Options.ts`)